app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsViewState.kt

```diff
@@ -4,14 +4,14 @@ import agency.five.codebase.android.movieapp.ui.component.ActorCardViewState
 import agency.five.codebase.android.movieapp.ui.component.CrewItemViewState
 
 data class MovieDetailsViewState(
-    val id: Int,
-    val imageUrl: String?,
-    val voteAverage: Float,
-    val title: String,
-    val overview: String,
-    val isFavorite: Boolean,
-    val crew: List<CrewmanViewState>,
-    val cast: List<ActorViewState>,
+    val id: Int = 0,
+    val imageUrl: String? = "",
+    val voteAverage: Float = 8.1.toFloat(),
+    val title: String = "",
+    val overview: String = "",
+    val isFavorite: Boolean = true,
+    val crew: List<CrewmanViewState> = listOf(),
+    val cast: List<ActorViewState> = listOf(),
```

Comment:
Hardcoding values here should be avoided since those values do not provide any real value to the app and could confuse other developers when maintaining the code. 

If you need these values for a preview just create a new object in a preview function and pass these hardcoded values in the preview function. 
Remove these hardcoded values here.

Reply:
If you need these values for an initial view state just follow my other comment with `EMPTY` pattern explained.